app/serializers/rest/web_push_subscription_serializer.rb

```diff
@@ -4,7 +4,7 @@ class REST::WebPushSubscriptionSerializer < ActiveModel::Serializer
   attributes :id, :endpoint, :alerts, :server_key
 
   def alerts
-    object.data&.dig('alerts') || {}
+    (object.data&.dig('alerts') || {}).each_with_object({}) { |(k, v), hash| hash.merge(k => ActiveModel::Type::Boolean.new.cast(v)) }
```

Comment:
Why not just `hash[k] = ActiveModel::Type::Boolean.new.cast(v)`?
The point of `each_with_object` is to mutate the object.